### ISSUE TITLE
Implement parsing Option<T> into Value and/or ValueRef

### ISSUE THREAD

**Issue by dowdw** [jgallagher/rusqlite#570]
I can't push my branch apparently, but the code is super simple.

In types/value.rs

```
impl <T> From<Option<T>> for Value
    where T: Into<Value> {
    fn from(v: Option<T>) -> Value {
        match v {
            Some(x) => x.into(),
            None => Value::Null,
        }
    }
}
```
In types/value_ref.rs
```
impl<'a, T> From<Option<T>> for ValueRef<'a> 
    where T: Into<ValueRef<'a>> {
    fn from(s: Option<T>) -> ValueRef<'a> {
        match s {
            Some(x) => x.into(),
            None => ValueRef::Null,
        }
    }
}
```

Passes all tests

**Comment by dowdw** [comment]
Figured out how the PR process works, its here: https://github.com/jgallagher/rusqlite/pull/571

**Comment by gwenn** [comment]
Fixed in 0.21.0.
